.github/workflows/security.yaml

```diff
@@ -0,0 +1,15 @@
+name: Security Scan
+
+on: [push, pull_request]
```

Comment:
We lint and test also on tags to version releases. Can we also do security scans there too?